check_esxi_hardware.py

```diff
@@ -659,10 +659,17 @@ def handler(signum, frame):
   try:
     conntest = pywbem.WBEMConnection(hosturl, (user,password))
     c = conntest.EnumerateInstances('CIM_Card')
-  except:
+  except pywbem.cim_http.Error, arg:
     #raise
```

Comment:
This comment should be simply removed

Reply:
You refer to `#raise` ? I'd like to keep it as it's a handy way to identify (potentially) unknown cim errors.